### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by elmar8287**
- Created a class called Solver.
- Created a method called factorial that takes one argument, an integer N, and returns the factorial for that number.
- Created a method called reverse that takes one argument, a string word, and returns word reversed.
- Created a method called fizzbuzz that takes one argument, an integer N, and returns a string.
- When N is divisible by 3, return "fizz".
- When N is divisible by 5, return "buzz".
- When N is divisible by 3 and 5, return "fizzbuzz".
- Any other case, return N as a string.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @elmar8287, @OLIPLICHE & @Donard97, 

**This is a great project you have going on, well-done lads**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+
+## Built With
+- Ruby
+- rubocop
+
+## Getting Started
+- Make sure you have `Ruby` installed in your system. You can install it [here](https://www.ruby-lang.org/en/documentation/installation/).
+- Clone this repository by running `git@github.com:Donard97/oop-school-library-app.git` in your command line.
+- Navigate to the repository by running `cd oop-school-library-app`.
+- Run the project by running `ruby main.rb` on the command line.
```
Comment (mke2111) [inline README.md]: - [x] Let us clean up this section to reflect this project and not the Library project.

Comment (OLIPLICHE) [reply README.md]: Thank you for letting us know.

**Review by EricMbouwe — APPROVED** [review]
### Hi @elmar8287, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OLIPLICHE** [comment]
Thank you

**Pull request merged**